package.json

```diff
@@ -22,10 +22,12 @@
   "engines": {
     "node": "8.9.4"
   },
-  "main": "src/index.js",
+  "main": "vast-client-node.js",
+  "module": "src/index.js",
+  "browser": "vast-client.js",
```

Comment:
Since we're changing this, maybe we should also rename those files as follows, in webpack build conf and package.json and in README / migration guide.

```
  "main": "dist/vast-client-node.js",
  "module": "src/index.js",
  "browser": "dist/vast-client-browser.js",
```

i.e. 1) move to `dist` and 2) optionally also append `-browser` in browser version. 
For people using automatic tools and `require('vast-client')`, this would non-breaking. This is breaking only for users who reference directly the filename. Anyway 2.0.0 is just out and probably no one is using it yet, so it's a good time to change if we want.

WDYT?

Reply:
Yes I agree with having a `dist` directory for the bundled files. And it should not be really considered a breaking change since the library is served as an `npm` package first. 👍